ui/src/app/pages/workspace/workspace-card.tsx

```diff
@@ -295,14 +295,19 @@ export const WorkspaceCard = fp.flow(withNavigation)(
                     Last Changed: {displayDate(workspace.lastModifiedTime)}
                   </div>
                 </FlexColumn>
-                <FlexColumn style={{justifyContent: 'flex-end', alignItems: 'flex-end', direction: 'rtl', marginLeft: 15}}>
-                  {accessTierShortName === AccessTierShortNames.Controlled && <ControlledTierBadge/>}
+                <FlexColumn style={{justifyContent: 'flex-end', marginLeft: '0.8rem'}}>
+                  <FlexRow style={{alignContent: 'space-between'}}>
+                    {adminLocked &&
+                    <FlexColumn data-test-id='workspace-lock' style={{justifyContent: 'flex-end'}}>
```

Comment:
Can this FlexColumn be a `<div>` instead, if we keep the style?

Reply:
I tried but it causes issue with icon alignment , is it ok if we can merge this and i can re-work on whole Flex thingy?

<img width="1618" alt="Screen Shot 2021-11-18 at 11 28 44 AM" src="https://user-images.githubusercontent.com/34481816/142457153-15184ce4-f6b3-4186-a621-b38689d8e096.png">

Reply:
ok